h2o-r/h2o-package/R/frame.R

```diff
@@ -4034,7 +4034,7 @@ use.package <- function(package,
             is.character(version), length(version)==1L,
             is.logical(use), length(use)==1L)
 
-   if (package=="data.table" && use) { # not sure if this is needed.  Keeping it for now.
+   if (package=="data.table" && use && "data.table" %in% rownames(installed.packages())) { # not sure if this is needed.  Keeping it for now.
```

Comment:
looks like `find.package` is much more efficient that listing `installed.packages`:
```
use && length(find.package("data.table", quiet=TRUE)) > 0
```

Reply:
can probably improvedfor the "but64" test below as well

Reply:
btw, should we not at least emit a warning if user is explicitely calling `use.package("data.table")` but the package is not installed? or is it only used internally?

Reply:
oh I see, it just returns a boolean

Reply:
Thanks for the tip!